Module_00/README.md

```diff
@@ -1,4 +1,67 @@
 # Setting up your environment
 
-## IDE
-## Terminal
+## Setup
+### Installing VSCode
+  - Mac/Apple
+      - https://code.visualstudio.com/docs/setup/mac
+  - Windows/PC
+    - https://code.visualstudio.com/docs/setup/windows
+
+### Installing Terminal Tools
+  - Mac/Apple
+    - https://www.iterm2.com/
+  - Windows/PC
+    - You will most likely need to install Git/Bash tool, please check in with one of your ThinkCERCAMP counselors to help out!
+***
+
+## So what the heck are we learning?
+  - Regardless of what language we are writing code in, the internet compiles what we write down to three building blocks of the web
+    - HTML
+    - CSS
+    - JavaScript (no relation to Java!)
+  - We use certain tools and "compilers" for the browser to then translate what we write into one, two, or all of those three
+  - Then those three get compiled down to even more technical machine code that our computers can read (all dem 0s and 1s)
+
+## Vocabulary
+  Term  | Definition
+  ----  | ----
+  Terminal/Command Line | The tool we use to access the skeleton/bones of our system. We can create, remove, move, edit, etc. files and folders all from within the terminal!
+  HTML  | Hypertext Markup Language
+  CSS | Cascading style sheets
+  JavaScript |  One of the foundational languages that the browser reads (along with HTML and CSS)
+  Markdown  | A language that allows us to mock/create files formatted such as this one without needing a formatting bar (bullet points, tables, bold, italics, etc...)
+  Directory | A folder!
+  Path | Either `relative` or `absolute` the path shows how to get from one file/folder to another
+  Code Snippet | A little 'blurb' of code.  Feel free to write multipline code snippets in SLACK by surrounding your lines with triple-ticks (to the left of the 1 key) or single ticks for single-line snippets (ex: `my first code snippet`)
+  ```
```

Comment:
the formatting of this line is a little strange, because you can't see the ticks in the README